src/Field/Field.java

```diff
@@ -3,40 +3,39 @@
 public class Field {
 
     private final Square[][] field;
+    private final MinePlacer minePlacer;
 
-    public Field(int row, int col) {
-        this.field = new Square[row][col];
-        for(int x = 0; x < row; x++) {
-            for(int y = 0; y < col; y++) {
+    public Field(Size size, MinePlacer minePlacer) {
+        int height = size.getHeight();
+        int width = size.getWidth();
+        this.minePlacer = minePlacer;
+        this.field = new Square[height][width];
```

Comment:
This is ok, but you could just put size.getHeight(), size.getWidth() in the square brackets